strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/testing/TestCaseWithMavenArtifactGenerationAndIndexing.java

```diff
@@ -212,15 +209,12 @@ public void reIndex(String storageId,
             throws IOException
     {
         Repository repository = configurationManagementService.getConfiguration()
-                                                              .getStorage(storageId)
-                                                              .getRepository(repositoryId);
+                .getStorage(storageId)
+                .getRepository(repositoryId);
```

Comment:
Cosmetic: Could you please restore the original indentation? :)

Reply:
yes
sorry for this